packages/modular-scripts/src/start.ts

```diff
@@ -107,6 +108,11 @@ async function start(packageName: string): Promise<void> {
     )}`,
   );
 
+  const reactVersion = externalResolutions?.['react'];
```

Comment:
nit: might be worth abstracting this check since I see it in two places - might be nice to be able to update/change this in 1 place

Reply:
Done.